pages/docs/index.tsx

```diff
@@ -156,6 +156,7 @@ const Docs = ({ bitbucket_auth_url, image_name }: { bitbucket_auth_url: string,
 			console.debug('[handleBuildButtonClick] /api/dpu/pubsub response:', response.data);
 			setBuildStatus(response.data);
 			if (response.data.success) {
+				setIsButtonDisabled(true); //If we get success as a status of the build, then the trigger button should be disabled after that. 
```

Comment:
This change seems redundant. The button is already disabled at this point. Why are we setting it to disabled again?

And does this even work? The `finally` clause will run after this and enable the button again. 

A better approach would be to move the `setIsButtonDisabled(false)` inside the catch block and remove this change as well as the `finally` block.